### ISSUE TITLE
remote data packages with relative paths to resources should not have a local_data_path

### ISSUE THREAD

**Issue by ndkv** [frictionlessdata/datapackage-py#97]
The `local_data_path()` function on [line 93](https://github.com/frictionlessdata/datapackage-py/blob/master/datapackage/resource.py#L93) in `resource.py` returns a mangled `path` [1] for remote data packages that use relative paths to their sources (see e.g. [this data package](https://raw.githubusercontent.com/ndkv/datasets/master/gemeenschappelijke_regelingen/partial/datapackage.json)).

A possible fix is to nullify `path` in case we are dealing with an URL:

```python
def local_data_path(self):
    '''str: The absolute local path for the data.'''

    path = self._absolute_path(self.descriptor.get('path'))
    if path:
        if not _is_url(path):
            return os.path.abspath(path)
        else:
            return None
```

I found this issue while looking for a solution to #63; at one point I noticed that the condition on [line 151](https://github.com/frictionlessdata/datapackage-py/blob/master/datapackage/datapackage.py#L151) will always fail for URLs due to the corrupted `local_data_path`. 

I'll proceed to implement this fix and write tests if you think its an agreeable solution. 

[1] `/absolute/path/https://remote/path`

**Comment by vitorbaptista** [comment]
@ndkv Thanks for the report.

I'm not sure if I understood the issue, though, but if the path is an URL, `local_data_path` should return None.

You seem to have a good grasp of the issue, so if you have the time, please go on and submit a pull request. Just make sure write a failing test that triggers this bug before actually solving it, so we're sure it doesn't creep back in the code.

**Comment by ndkv** [comment]
@vitorbaptista My first sentence is somewhat convoluted, indeed. Sorry about that.  

Let me try again: `local_data_path()` turns an URL into an absolute path which results the borked path `/absolute/path/https://remote/path`. This in turn makes `safe()` in `datapackage.py` flag the URL as unsafe.

I submitted a pull request; I'd love some feedback if anything is off. Else I'll celebrate my second pull request to an OS project. 😁
